onet.go

```diff
@@ -18,4 +18,9 @@ ProtocolInstance.
 package onet
 
 // Version of onet.
-const Version = "1.2"
+const Version = "1.3"
```

Comment:
As it's breaking, I would not put it as a `v1`-version, but rather a `v2-alpha` or so.